src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -306,8 +306,7 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
             try {
                 $service = $this->$method();
             } catch (\Exception $e) {
-                unset($this->loading[$id]);
-                unset($this->services[$id]);
+                unset($this->loading[$id], $this->services[$id]);
```

Comment:
:+1:  for consistency, than it can be set in the configuration of the fixer and we won't have PR's for it any more :)